src/formatting/syntux/session/ignore_path.rs

```diff
@@ -60,4 +60,35 @@ mod test {
             _ => {}
         };
     }
+    #[test]
+    fn test_ignore_path_set_with_dir() {
+        match option_env!("CFG_RELEASE_CHANNEL") {
+            // this test requires nightly
+            None | Some("nightly") => {
```

Comment:
I know the other test here is using this pattern, but we've moved towards leveraging the below pattern elsewhere in tests that are nightly-only

```rust
if !is_nightly_channel!() {
    return;
}
```

Would you mind updating this (and the one above) to follow this pattern? `is_nightly_channel` is available at the crate root. Using this helps keeps things a bit more encapsulated, and also helps reduce some of the indentation levels

Reply:
I'll follow this pattern!